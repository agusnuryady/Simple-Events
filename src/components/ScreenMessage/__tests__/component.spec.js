import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import ScreenMessage from '..';

describe('Some component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ScreenMessage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
