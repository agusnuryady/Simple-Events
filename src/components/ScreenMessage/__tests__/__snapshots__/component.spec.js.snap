// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Some component renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flex": 1,
        "height": "100%",
        "justifyContent": "center",
        "padding": 32,
        "width": "100%",
      },
      Object {},
    ]
  }
>
  <View
    style={
      Object {
        "marginBottom": 12,
      }
    }
  >
    <View />
  </View>
  <View />
</View>
`;
